runtime/build.gradle

```diff
@@ -20,8 +20,6 @@ targetList.each { targetName ->
         dependsOn "${targetName}Debug"
         dependsOn "${targetName}Release"
         target targetName
-	    if (!isWindows())
-	       compilerArgs '-fPIC'
```

Comment:
So `-fPIC` is not needed for all targets, not just mingw?
It also present in 4 more files:
```
backend.native/build.gradle
backend.native/tests/build.gradle
libclangext/build.gradle
llvmDebugInfoC/build.gradle
```

Reply:
Guess it just not needed here.